retrieval/ena-project.rst

```diff
@@ -29,6 +29,8 @@ Once the page has fully loaded, the project should look like this:
 .. image:: images/project-record.png
    :align: center
 
+.. _nav box:
```

Comment:
Have not seen this type of linking used before but this is working correctly
.. _nav box: (location)
:ref:`the navigation box <nav box>`. (reference)

Reply:
Yeah! Custom anchors - pretty handy! 🙂